bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,352 @@
+For updates go to official repository at [https://github.com/souliss/bindingopenhab2](https://github.com/souliss/bindingopenhab2)
+
```

Comment:
Can this be removed? As this is an opensource project, we only want to be promoting links to merged bindings that are community maintained.